pkg/core/transaction/input.go

```diff
@@ -34,3 +35,9 @@ func (in *Input) EncodeBinary(w io.Writer) error {
 	}
 	return nil
 }
+
+// Size returns the size in bytes of the Input
+func (in *Input) Size() int {
+	var ui16 uint16
+	return in.PrevHash.Size() + int(unsafe.Sizeof(ui16))
```

Comment:
in.PrevHash.Size() + 2 // 2 = sizeOf uint16

Reply:
:+1: